src/cosmosdb-preview/azext_cosmosdb_preview/_params.py

```diff
@@ -331,3 +331,37 @@ def load_arguments(self, _):
         c.argument('account_name', account_name_type, id_part=None, required=True, help='Name of the CosmosDB database account')
         c.argument('database_name', database_name_type, required=True, help='Name of the mongoDB database')
         c.argument('container_name', options_list=['--name', '-n'], required=True, help='Name of the mongoDB collection')
+
+    # Sql container partition retrieve throughput
+    with self.argument_context('cosmosdb sql container retrieve-partition-throughput') as c:
+        c.argument('account_name', account_name_type, id_part=None, required=True, help='Name of the CosmosDB database account')
+        c.argument('database_name', database_name_type, required=True, help='Name of the CosmosDB database name')
+        c.argument('container_name', options_list=['--name', '-n'], required=True, help='Name of the CosmosDB container')
+        c.argument('physical_partition_ids', options_list=['--physical-partition-ids', '-p'], required=False, help='comma separated list of physical partition ids')
+        c.argument('all_partitions', arg_type=get_three_state_flag(), help="switch to retrieve throughput for all physical partitions")
+
+    # Sql container partition redistribute throughput
+    with self.argument_context('cosmosdb sql container redistribute-partition-throughput') as c:
+        c.argument('account_name', account_name_type, id_part=None, required=True, help='Name of the CosmosDB database account')
+        c.argument('database_name', database_name_type, required=True, help='Name of the CosmosDB database name')
+        c.argument('container_name', options_list=['--name', '-n'], required=True, help='Name of the CosmosDB collection')
+        c.argument('evenly_distribute', arg_type=get_three_state_flag(), help="switch to distribute throughput equally among all physical partitions")
+        c.argument('target_partition_info', options_list=['--target-partition-info'], required=False, help="information about desired target physical partition throughput eg: '[{\"id\":0,\"throughput\":1200},{\"id\":1,\"throughput\":1200}]'")
+        c.argument('source_partition_info', options_list=['--source-partition-info'], required=False, help="information about source physical partition ids eg: '[{\"id\":2}]'")
```

Comment:
Use `action` to parse complicated parameter instead of asking customers to pass in json string which is not user friendly